x/gov/migrations/v3/convert.go

```diff
@@ -46,6 +48,9 @@ func ConvertToLegacyProposal(proposal v1.Proposal) (v1beta1.Proposal, error) {
 	if err != nil {
 		return v1beta1.Proposal{}, err
 	}
+	if len(msgs) != 1 {
+		return v1beta1.Proposal{}, sdkerrors.ErrInvalidType.Wrap("can't convert a gov/v1 Proposal to gov/v1beta1 Proposal when amount of proposal messages is more than one")
+	}
 	for _, msg := range msgs {
```

Comment:
The for loop isn't needed anymore

Reply:
yes... too late :/ Is it worth to create another PR?